docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: "3"
+services:
+  php:
+    build:
+      context: .
+      args:
+        - PHP_VERSION=${TRAVIS_PHP_VERSION:-7.2}
+      dockerfile: Dockerfile
+    depends_on:
+      - mongo
+    volumes:
+      - ./:/docker
+    environment:
+      - MONGO_HOST=mongo
```

Comment:
Running locally the environment variables are correct.  The "default" ones are `localhost`/`27017`.  We cannot move these to `.travis.yml` only because the local Docker needs them too.

Reply:
👍

Reply:
Additionally this Docker configuration is correct for the docker-compose and the current project.  Moving the docker connect information to environment variables means we're encouraging the user to setup their own hardware to run the repository which is fine if they *don't* want to use Docker.